oas_worker/app/jobs/jobs.py

```diff
@@ -168,3 +169,24 @@ def asr_mock(ctx, args):
     meta = { "mock": "yes" }
     return { "patches": patches, "meta": meta }
 
+
+# Devset Evaluation Util
+@worker.job(name="cba2oas_id")
```

Comment:
Why is this a job? If I read things correctly this just executes a single query. This should better be done directly through the REST API or helper function, no need for this to be in a job.